vizro-core/src/vizro/actions/_actions_utils.py

```diff
@@ -108,40 +83,16 @@ def _get_parent_vizro_model(_underlying_callable_object_id: str) -> VizroBaseMod
     )
 
 
-def _apply_table_filter_interaction(
-    data_frame: pd.DataFrame, target: str, ctd_filter_interaction: Dict[str, CallbackTriggerDict]
-) -> pd.DataFrame:
-    ctd_active_cell = ctd_filter_interaction["active_cell"]
-    ctd_derived_viewport_data = ctd_filter_interaction["derived_viewport_data"]
-    if not ctd_active_cell["value"] or not ctd_derived_viewport_data["value"]:
-        return data_frame
-
-    # ctd_active_cell["id"] represents the underlying table id, so we need to fetch its parent Vizro Table actions.
-    source_table_actions = _get_component_actions(_get_parent_vizro_model(ctd_active_cell["id"]))
-
-    for action in source_table_actions:
-        if action.function._function.__name__ != "filter_interaction" or target not in action.function["targets"]:
-            continue
-        column = ctd_active_cell["value"]["column_id"]
-        derived_viewport_data_row = ctd_active_cell["value"]["row"]
-        clicked_data = ctd_derived_viewport_data["value"][derived_viewport_data_row][column]
-        data_frame = data_frame[data_frame[column].isin([clicked_data])]
-
-    return data_frame
-
-
 def _apply_filter_interaction(
     data_frame: pd.DataFrame, ctds_filter_interaction: List[Dict[str, CallbackTriggerDict]], target: str
 ) -> pd.DataFrame:
     for ctd_filter_interaction in ctds_filter_interaction:
-        if "clickData" in ctd_filter_interaction:
-            data_frame = _apply_graph_filter_interaction(
-                data_frame=data_frame, target=target, ctd_filter_interaction=ctd_filter_interaction
-            )
-
-        if "active_cell" in ctd_filter_interaction and "derived_viewport_data" in ctd_filter_interaction:
-            data_frame = _apply_table_filter_interaction(
-                data_frame=data_frame, target=target, ctd_filter_interaction=ctd_filter_interaction
+        if "modelID" in ctd_filter_interaction:
```

Comment:
Do we need this statement (`if "modelID" in ctd_filter_interaction:`) at all?

Reply:
This was supposed to make the code more stable in case of custom components. Imagine someone making a custom model that doesn't require filtering functionality, so he chooses not to even create those callback required objects, in this case this function then does not fail. As far as I am aware the `_get_filtered_data` function is always called, so this would be necessary

Reply:
`ctds_filter_interaction` contains info only for the components that have defined `filter_interaction`. Actually, only for components that: `hasattr(triggered_model, "_get_figure_interaction_input")`. See changes inside the function: `_get_inputs_of_figure_interactions`. 

However, we can leave this statement (`if "modelID" in ctd_filter_interaction:`), since it will probably be refactored in the actions V2 soon.

Reply:
Hm I see what you mean. 2 questions:
- do we prefer to fail if a user overwrites this method without adding the `modelID`
- or do we prefer the code not to fail, but the filtering also not to happen

As far as I understand, the answer to the above determines if we should leave the statement in or not. We should definitely not leave it in just because things are getting refactored soon

Reply:
If we can't perform filter_interaction without `modelID` part of the input, it means we need to raise an exception or some kind of warning to the user. It really all depends again on the fact how do we deal with "actions_info", what is the part of the "actions_info" configuration, and what is the part of the "_actions_utils" functions. Does it makes sense to continue this conversation in the https://github.com/mckinsey/vizro/pull/289#discussion_r1495590983?

Reply:
ok dokes!